src/super_gradients/conversion/conversion_utils.py

```diff
@@ -12,7 +12,7 @@
     (torch.int16, np.int16),
     (torch.int8, np.int8),
     (torch.uint8, np.uint8),
-    (torch.bool, np.bool),
+    # (torch.bool, np.bool),
```

Comment:
?

Reply:
Yeah, I'll take it out! I just had an error locally and didn't have internet to update my versions, so I commented out and accidentally added it to the PR

Reply:
Removed 
8d654eb